dynamic_listings/python/default/collisiondetection/AABB.py

```diff
@@ -26,23 +24,23 @@ def __init__(self, point: Point, width: int, height: int) -> None:
         :height: The rectangle Height
 
         """
-        self._point: Point = point
-        self._width: int = width
-        self._height: int = height
+        self.corner = point
+        self.width = width
+        self.height = height
 
 
-def rect_rect_collision(A: Rectangle, B: Rectangle):
+def rect_rect_collision(A: Rectangle, B: Rectangle) -> bool:
     """
     Checks for an axis-aligned bounding rectangle collision
 
     :A: The First Rectangle
     :B: The Second Rectangle
     :returns: A boolean that tells if the rectangles collided
     """
-
-    if (A.corner.x < B.corner.x + B.width) and\
+    
+    is_colliding = (A.corner.x < B.corner.x + B.width) and\
        (A.corner.x + A.width > B.corner.x) and\
        (A.corner.y < B.corner.y + B.height) and\
-       (A.corner.y + A.height > A.corner.y):
-        return True
-    return False
+       (A.corner.y + A.height > A.corner.y)
```

Comment:
I think you can omit backslashes in parentheses, it should look cleaner. Good call on stuffing the complex statement in a variable.